figures/drawing-slice-2.fig

```diff
@@ -0,0 +1,36 @@
+                             AS 1000   ◄────BGP───► AS 65000
+
+ ┌───┐                     ┌────────┐                    ┌───┐
+ │NF1│       172.30.1/30   │        │   172.30.2/30      │NF2│
+ └─┬─┘   ┌───┐          ┌──┴┐      ┌┴──┐          ┌───┐  └─┬─┘
+   │     │   │.1      .2│   │      │   │.2      .1│   │    │
+ ──┴─────┤GW1│----------│PE1│      │PE2│----------│GW2├────┴──
+         │   │ vlan-id  │   │      │   │ vlan-id  │   │
+         └───┘   100    └──┬┘      └┬──┘   200    └───┘
+192.168.100/24             │        │             192.168.200/24
```

Comment:
Idem. Please use one of the addresses in https://datatracker.ietf.org/doc/rfc5737/. Thanks.